nbuild/02.build_manifests.md

```diff
@@ -0,0 +1,154 @@
+# 2 Build Manifests
+A build manifest is a set of instructions that fetches, compiles and splits a software into one or more packages. It is typically made of three main parts:
+* A set of metadata used as a reference for the metadata of the built packages
+* A list of versions and a corresponding set of data, one per version, called the versionized arguments of a build
+* A list of instructions (taking the form of a python function) that builds the packages
+
+## 2.1 Writing a build manifest
+### 2.1.1 The `@manifest` decorator
+The `@manifest` decorator requires several parameters:
+* `name` is the name of the software being built.
+* `category` is the category the built packages belong to.
+The category should be in `snake-case`.
+* `description` is a short description of what this package is for, what it does, etc.
+It should start with an uppercase letter and finish with a dot.
+* `tags` are a list of tags helping a user to find the built packages easily.
+Each tag should be in `snake-case`.
+* `maintainer` is the email address of the maintainer of this build manifest.
+* `licenses` are the software license(s) of this package.
+* `upstream_url` is the URL pointing to the home page of the software.
+* `kind` is the kind of packages
+* `versions_data` is the versionized list of arguments.
+
+#### 2.1.1.1 Effective and virtual packages
+Two kinds of packages are possible: **effective** and **virtual**.
+
+A **Virtual** package is a package with no data, only listing its dependencies.
+
+An **Effective** package is, on the contrary, a package with data.
```

Comment:
"on the contrary" sounds weird

Reply:
what about this?
```suggestion
On the opposite, an **Effective** package is a package with data.
```

Reply:
```suggestion
Conversly, an **Effective** package is a package with data.
```
Still sounds kind of odd, but a little better I think.